django_libsass.py

```diff
@@ -42,7 +54,12 @@ def compile(**kwargs):
         INCLUDE_PATHS = get_include_paths()
 
     kwargs = kwargs.copy()
-    kwargs['include_paths'] = (kwargs.get('include_paths') or []) + INCLUDE_PATHS
+    kwargs['include_paths'] = kwargs.get('include_paths', []) + INCLUDE_PATHS
```

Comment:
This will fail if `None` or `''` is passed as the `include_paths` kwarg. Off the top of my head, I don't know if anything in django-compressor actually does this, but I'd prefer not to risk breaking anything.